src/features/example/screens/ExampleScreen/ExampleScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, memo, useEffect, useState } from "react";
+
+import TableView from "components/TableView/TableView";
+import { Rows } from "components/TableView/TableView.types.example";
+
+import { userApi } from "../../api/example.api";
+
+interface UserData {
+  page: number;
+  per_page: number;
+  total: number;
+  total_pages: number;
+}
+
+const ExampleScreen: FC = () => {
+  const [rows, setRows] = useState<Rows[]>([]);
+  const [rowsPerPage, setRowsPerPage] = useState<number>(0);
+  const [page, setPage] = useState<number>(1);
+  const [count, setCount] = useState<number>(0);
+
+  const columns = [
```

Comment:
This variable value doesn't change during component lifecycle so I think we should move this variable to out of the component for better performance.

Reply:
Or we can use useEffect to prevent re-declaration again and again

Reply:
Working on it bfc75c0

Reply:
Done 6d463a0

Not sure if it makes big difference removing the `useState`s but managed to refactor the code removing most of them